.github/workflows/scripts/start-acs.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+set -euox pipefail
+
+
+cd $STACKROX_DIR
+
+echo "GITHUB_OUTPUT= ${GITHUB_OUTPUT}"
+./deploy/k8s/central.sh
+kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
+sleep 20
+
+./deploy/k8s/sensor.sh
+
+kubectl -n stackrox set env deploy/sensor MUTEX_WATCHDOG_TIMEOUT_SECS=0 ROX_FAKE_KUBERNETES_WORKLOAD=long-running ROX_FAKE_WORKLOAD_STORAGE=/var/cache/stackrox/pebble.db
+kubectl -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+
+kubectl -n stackrox set env deploy/central MUTEX_WATCHDOG_TIMEOUT_SECS=0
+kubectl -n stackrox patch deploy/central -p '{"spec":{"template":{"spec":{"containers":[{"name":"central","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+
+max_tries=200
+tries=0
+while [ "$tries" -lt "$max_tries" ]; do
+    ready="$(kubectl -n stackrox get pod -l app=central -o jsonpath='{.items[*].status.containerStatuses[?(@.ready == true)]}')"
+
+    if [[ -n "$ready" ]]; then
+	    echo "Central is running"
+	    break
+    fi
+
+    tries=$((tries + 1))
+    sleep 1
+done
+
+sleep 60
+
+ROX_ADMIN_PASSWORD=$(cat deploy/k8s/central-deploy/password)
+#echo "::add-mask::$ROX_ADMIN_PASSWORD"
```

Comment:
I wasn't able to pass the password to the other jobs without commenting this out. How can I pass the password, while keeping it secret?

Reply:
https://github.com/orgs/community/discussions/25225#discussioncomment-3246942

Save the masked password as a step and job output. You'll be able to reference this in other jobs with needs, like https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs#example-defining-outputs-for-a-job. 

How to mask the secret: 
```
echo "::add-mask::$ROX_ADMIN_PASSWORD"
echo "::set-output name=rox-password::${ROX_ADMIN_PASSWORD}" >> $GITHUB_OUTPUT
```

Reply:
How would this be impacted by the work you did in moving the cluster creation to stackrox/actions? I guess I will probably have to move what I have done to there as well. I am not clear how that will work.